Documentation/ceph-upgrade.md

```diff
@@ -476,15 +486,17 @@ located.
 ```yaml
   env:
     - name: ROOK_CSI_CEPH_IMAGE
-        value: "quay.io/cephcsi/cephcsi:v1.2.2"
+        value: "quay.io/cephcsi/cephcsi:v2.0.0"
```

Comment:
Instead of modifying this section, I really think we need a new section that is just dedicated to the optional step of updating to the CSI 2.0 driver. This upgrade doc really represents upgrades from Rook v1.1 to v1.2 since we'll be backporting it. We can change the doc later when we are ready to support the v1.2 to v1.3 upgrade. (See suggestion above)